labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -27,7 +33,7 @@ object ListManipulationExercise02 {
    * method above
    */
   def sumOfMany(l: List[Int]*): List[Int] = {
-    error("fix me")
+    l.reduce((l1,l2) => sumOfTwo(l1,l2))
```

Comment:
This function confused me a bit, but apparently l.reduce(sumOfTwo) on its own is enough, probably since Haskell knows that sumOfTwo is going from a pair of lists to a single list.